docs/src/fts_tantivy.md

```diff
@@ -0,0 +1,163 @@
+# Full-text search (Tantivy-based FTS)
+
+LanceDB also provides support for full-text search via [Tantivy](https://github.com/quickwit-oss/tantivy), allowing you to incorporate keyword-based search (based on BM25) in your retrieval solutions.
+
+The tantivy-based FTS is available in only Python, for the other languages you can use the [native FTS](fts.md).
```

Comment:
Maybe also mention here that Tantivy doesn't support object stores or incremental indexing either.